axelor-stock/src/main/resources/views/Inventory.xml

```diff
@@ -47,7 +47,11 @@
             <field name="productCategory" onSelect="action-inventory-on-select-product-category" onChange="action-inventory-on-change-product-category" grid-view="product-category-grid" form-view="product-category-form"/>
             <field name="excludeOutOfStock" colSpan="3"/>
             <field name="includeObsolete" colSpan="3"/>
-            <spacer/>
+           	<panel title="Racks range" colSpan="12">
+		      <field name="fromRack" colSpan="3"/>
+		      <field name="toRack" colSpan="3"/>
+		    </panel>
+		    <spacer/>
             <button name="fillInventory" title="Fill inventory" showIf="stockLocation != null" onClick="action-group-stock-inventory-fillinventory-click" colSpan="3"/>
```

Comment:
replace original <spacer> with this new subpanel and no needs to provide any colspan.

Reply:
My mistake, I typed 'original' instead of 'spacer' , keep main panel but replace spacer with sub panel for 'Rack range'